k8s-cleanup.tf

```diff
@@ -28,7 +28,7 @@ resource "kubernetes_deployment" "beekeeper_cleanup" {
   }
 
   spec {
-    replicas = var.k8s_cleanup_name_replicas
+    replicas = 1
```

Comment:
Done this because I'm also not sure of the implications of multiple cleanups, multiple path cleaners is _probably_ fine - though untested. But multiple cleanups editing the database simultaneously could be problematic.

Reply:
maybe aline of comment in the tf, that setting that value > 1 is untested?